### DISCUSSION TITLE
Need help with Installing for the MAC version.

### DISCUSSION THREAD

**Discussion by DiscountSniper** [Dpeta/pesterchum-alt-servers#40] [Q&A]
All the videos about installing the thing are outdated and no longer works. I cannot directly help with the issue due to the fact that I use a Windows laptop than a MAC. 

We'd disable the the "Unidentified Developer" problem, but when we finally try to install Pesterchum, it just doesn't work and the system wants to throw it away.

What do we do?

**Comment by Dpeta** [comment]
Hmm, well, if the build doesn't work you can always try running one of the windows builds with [wine](https://www.winehq.org/). Or you could consider installing python, downloading the repository, and just running pesterchum.py directly.
